src/components/Router.js

```diff
@@ -2,13 +2,13 @@
 import React from 'react';
 
 // third party library
-import { Switch, Route, Router  } from 'react-router-dom';
+import { Switch, Route, Router } from 'react-router-dom';
```

Comment:
#### Nitpick
When commenting imports it's actually one of the following (though this is not a comprehensive list)
```js
// react libraries
...react packages

// third-party libraries 
....other installed packages

// components
....components 

// actions
...


```

So, remove the Moduler importations stated on line 7 and in every other file.

Reply:
Fixed